docs/pages/repo/docs/handbook/deploying-with-docker.mdx

```diff
@@ -155,3 +155,27 @@ COPY --from=installer --chown=nextjs:nodejs /app/apps/web/.next/static ./apps/we
 
 CMD node apps/web/server.js
 ```
+
+## Remote caching
+
+To take advantage of remote caches during Docker builds, you will need to make sure your build container has the proper credentials to access your Remote Cache.
```

Comment:
Not a blocking issue, but I'm not a fan of the word "proper", because:

- it's redundant (there is no case for setting up credentials  _im_properly)
- feels preachy ("be good now" "you need to do this right, or else!")